src/main/java/com/psj/itembrowser/order/service/OrderCalculationResult.java

```diff
@@ -18,16 +22,57 @@
  * -----------------------------------------------------------
  * 2024-02-12        ipeac       최초 생성
  */
-@Data
-@NoArgsConstructor(access = AccessLevel.PROTECTED)
-@AllArgsConstructor
-public class OrderCalculationResult implements Serializable {
-	private double totalPrice;
-	private double totalDiscount;
-	private double shippingFee;
-	private double orderTotal;
-
-	public static OrderCalculationResult of(double totalPrice, double totalDiscount, double shippingFee, double orderTotal) {
-		return new OrderCalculationResult(totalPrice, totalDiscount, shippingFee, orderTotal);
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED, force = true)
+public final class OrderCalculationResult implements Serializable {
+	private final BigDecimal totalPrice;
+	private final BigDecimal totalDiscount;
+	private final Long shippingFee;
+	private final BigDecimal totalNetPrice;
```

Comment:
force 옵션과 final 설정으로 해당 클래스, 클래스 속성값이 불변이라는 의미를 부여하고 싶었습니다.

force = true 의 경우
(소스코드 lombok)

```java
	/**
	 * If {@code true}, initializes all final fields to 0 / null / false.
	 * Otherwise, a compile time error occurs.
	 * 
	 * @return {@code} true to force generation of a no-args constructor, picking defaults if necessary to assign required fields.
	 */
	boolean force() default false;
```

로서 final 필드는 컴파일 시점에 강제로 기본값으로 필드 속성이 채워집니다.